osgar/drivers/logzeromq.py

```diff
@@ -88,7 +88,7 @@ def run_reqrep(self):
                 socket.send_string(request)
                 while self.bus.is_alive():
                     try:
-                        resp = socket.recv()
+                        resp = socket.recv().decode('ascii')
```

Comment:
why this change? `bytes` are preferred to `str`

Reply:
I changed my mind as there is already string used in request part (`socket.send_string(request)` 3 lines above). In general request can be string and response binary blob (for example image), but let's ignore it for the moon moment.